devinfo/src/mountinfo/io_utils.rs

```diff
@@ -0,0 +1,27 @@
+use crate::mountinfo::error::{MountInfoError, Result};
+use std::{fs::read, path::Path};
+
+const DEFAULT_RETRY_COUNT: u32 = 2;
+
+pub(crate) fn consistent_read<P: AsRef<Path>>(
+    path: P,
+    retry_count: Option<u32>,
```

Comment:
You are specifying Some(2), does this need to be optional?

Reply:
Sure, why not..

Reply:
Added the `SafeMountIter::get_with_retries(retry_count)` API, with takes an optional argument for the retry-count.
Changed `SafeMountIter::get()` to use `SafeMountIter::get_with_retries()` so that it reverts to the default retry count of 2 when that API is used.